### ISSUE TITLE
[web: a11y] unable to inspect items under heading using NVDA

### ISSUE THREAD

**Issue by yjbanov** [flutter/flutter#82463]
## Steps to Reproduce

1. Enable NVDA on Windows 10
2. Navigate to gallery.flutter.dev
3. Enable accessibility
4. Use the Down Arrow key to reach "Material", "Cupertino", or "Styles & Other" headings

**Expected results:**

Once on a heading you should somehow be able to enter the list and inspect the items.

**Actual results:**

NVDA skips over the items to the next heading, e.g. from "Material" to "Cupertino".

This is not a problem with VoiceOver on macOS. You can use Control + Option + Shift + Down Arrow to enter the list under the heading. However, there does not seem to exist such a shortcut in NVDA.

**Comment by yjbanov** [comment]
Fixed by https://github.com/flutter/engine/pull/50794
